src/usr/sbe/HBconfig

```diff
@@ -28,3 +28,9 @@ config NO_SBE_UPDATES
     default n
     help
         Don't update the SBE SEEPROMs at all.
+
+config SBE_UPDATE_PROTECT_SEEPROM
+    default n
```

Comment:
This might need to be "y if SECUREBOOT" but I don't recall if all of those options have been ported forward to P9 yet or are only in the P8 branch.